mingw-w64-git/PKGBUILD

```diff
@@ -41,7 +41,23 @@ build() {
   export PYTHON_PATH=/usr/bin/python2
   cd "$srcdir/git"
 
-  make -j1 all
+  make -j1 all &&
+
+  gcc -DMAGIC_RESOURCE -mwindows -s -O2 -Wall -o edit-res.exe \
+	compat/win32/git-wrapper.c git.res -lshlwapi &&
+  cp edit-res.exe git-bash.exe &&
+  strip git-bash.exe &&
```

Comment:
Actually, it might wirk without this `strip` call together. The idea was to remove the symbols before editing the resources, but still being able to run `edit-res.exe` in `gdb`. But that only resulted in a segmentation fault in a later Pacman step.